README.md

```diff
@@ -1,39 +1,48 @@
-# RipMe 
+# RipMe
+
 [![Licensed under the MIT License](https://img.shields.io/badge/License-MIT-blue.svg)](/LICENSE.txt)
 [![Join the chat at https://gitter.im/RipMeApp/Lobby](https://badges.gitter.im/RipMeApp/Lobby.svg)](https://gitter.im/RipMeApp/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Subreddit](https://img.shields.io/badge/discuss-on%20reddit-blue.svg)](https://www.reddit.com/r/ripme/)
 ![alt Badge Status](https://github.com/ripmeapp2/ripme/actions/workflows/gradle.yml/badge.svg)
-[![Coverage Status](https://coveralls.io/repos/github/RipMeApp/ripme/badge.svg?branch=master)](https://coveralls.io/github/RipMeApp/ripme?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/RipMeApp/ripme/badge.svg?branch=main)](https://coveralls.io/github/RipMeApp/ripme?branch=main)
+
+The current active development repo for RipMe is located at [ripmeapp2/ripme](https://github.com/ripmeapp2/ripme/).
+
+RipMe has been maintained with ♥️ and in our limited free time by the following
+people, roughly in order from most recent primary developer, with current
+activity marked by color of the indicator:
 
-RipMe is maintained with ♥️ and in our limited free time by **[@MetaPrime](https://github.com/metaprime)**, **[@cyian-1756](https://github.com/cyian-1756)** and **[@kevin51jiang](https://github.com/kevin51jiang)**. If you'd like to contribute but aren't good with code, help keep us happy with a small contribution! Chat on [gitter](https://gitter.im/RipMeApp/Lobby).
+- **[@soloturn](https://github.com/soloturn)** 🟢,
+- **[@cyian-1756](https://github.com/cyian-1756)** 🟥,
+- **[@kevin51jiang](https://github.com/kevin51jiang)** 🟥,
+- **[@MetaPrime](https://github.com/metaprime)** 🟡,
+- and its original creator, **[@4pr0n](https://github.com/4pr0n)** 🟥.
 
```

Comment:
yellow probably is a better for my involvment given my delays ...

Reply:
Well, it's all relative. You can be reached in a couple of days. I disappeared for 6 years and am back "for now"